src/wolfio.c

```diff
@@ -800,6 +800,11 @@ int wolfIO_Send(SOCKET_T sd, char *buf, int sz, int wrFlags)
         nfds = (int)sockfd + 1;
     #endif
 
+        if (*sockfd >= FD_SETSIZE) {
```

Comment:
This looks good. Some platforms the SOCKET_T is an int... for those might also be worth adding `*sockfd < 0`.

Reply:
Oh yeah `sockfd` has already been dereferenced. Should be just `sockfd` here.

Reply:
Yep... Fixed